app/services/cached_commodity_service.rb

```diff
@@ -64,7 +64,7 @@ class CachedCommodityService
         {
           measure_condition_components: [
             { measurement_unit: %i[measurement_unit_description measurement_unit_abbreviations] },
-            :measure_condition,
+            { measure_condition: { measure: :measure_type } },
```

Comment:
What is your plan in terms of regression testing this? Making sure nothing has broken. Pretty tough to test all cases - but I guess if our smoke tests + unit tests - should be ok?

Reply:
My (assumption) was that there's no behavioural change so no change to specs. its an internal detail (does it eager load or not?)

I'm guessing there's not any existing mechanism to test for N+1s or these would already be getting picked up? I think it should be possible to include some kind of logging of query counts but actual N+1 detection I'm not sure about?

Reply:
Makes sense!